### DISCUSSION TITLE
how to plot OCV versus time

### DISCUSSION THREAD

**Discussion by alirahdarian** [pybamm-team/PyBaMM#4059] [General]
Hi,

Can someone tell me how can plot open circuit potential (OCV) over time in this code? I don't want to see different contributions in voltage. I just want to plot OCV versus time. (I could not find OCV in the output parameters)

import pybamm
model = pybamm.lithium_ion.DFN()
sim = pybamm.Simulation(model)
sim.solve([0, 3600])
output_variables = ["Voltage [V]"]
sim.plot(output_variables=output_variables)

**Comment by kawaMANMI** [comment]
model.variable_names() will show you all the output variables including 'Battery open-circuit voltage [V]'.  e.g.
sim.plot(output_variables=["Current [A]",
         "Electrolyte concentration [mol.m-3]", 'Battery open-circuit voltage [V]'])
